tools/splat.yaml

```diff
@@ -925,280 +926,333 @@ segments:
     files:
     - [0x415D90, c]
   - [0x4200C0, bin] # todo split this further
-  - name: battle/section_0/
+  - name: battle/area_kmr_part_1/
     type: code
     start: 0x4309A0
     vram: 0x80218000
+    overlay: True
     files:
-    - [0x4309A0, bin]
-  - name: battle/section_1/
+    - [0x4309A0, c]
+    - [0x431B80, .data, battle/area_kmr_part_1/battles]
+    - [0x431FB0, bin, battle/area_kmr_part_1/goomba]
+    - [0x4398A0, .rodata, battle/area_kmr_part_1/battles]
+    - [0x439984, bin]
+  - name: battle/area_kmr_part_2/
     type: code
     start: 0x439A90
     vram: 0x80218000
+    overlay: True
     files:
     - [0x439A90, bin]
-  - name: battle/section_2/
+  - name: battle/area_kmr_part_3/
     type: code
     start: 0x447440
     vram: 0x80218000
+    overlay: True
     files:
     - [0x447440, bin]
-  - name: battle/section_3/
+  - name: battle/area_mac/
     type: code
     start: 0x464490
     vram: 0x80218000
+    overlay: True
     files:
     - [0x464490, bin]
-  - name: battle/section_4/
+  - name: battle/area_hos/
     type: code
     start: 0x47F0B0
     vram: 0x80218000
+    overlay: True
     files:
     - [0x47F0B0, bin]
-  - name: battle/section_5/
+  - name: battle/area_nok/
     type: code
     start: 0x48D1A0
     vram: 0x80218000
+    overlay: True
     files:
     - [0x48D1A0, bin]
-  - name: battle/section_6/
+  - name: battle/area_trd_part_1/
     type: code
     start: 0x4A10A0
     vram: 0x80218000
+    overlay: True
     files:
     - [0x4A10A0, bin]
-  - name: battle/section_7/
+  - name: battle/area_trd_part_2/
     type: code
     start: 0x4B11F0
     vram: 0x80218000
+    overlay: True
     files:
     - [0x4B11F0, bin]
-  - name: battle/section_8/
+  - name: battle/area_trd_part_3/
     type: code
     start: 0x4CF1B0
     vram: 0x80218000
+    overlay: True
     files:
     - [0x4CF1B0, bin]
-  - name: battle/section_9/
+  - name: battle/area_iwa/
     type: code
     start: 0x4D01F0
     vram: 0x80218000
+    overlay: True
     files:
     - [0x4D01F0, bin]
-  - name: battle/section_10/
+  - name: battle/area_sbk/
     type: code
     start: 0x4DA5E0
     vram: 0x80218000
+    overlay: True
     files:
-    - [0x4DA5E0, bin]
-  - name: battle/section_11/
+    - [0x4DA5E0, bin] # 1 func
```

Comment:
any particular reason not to mark these as .text do splat can split them out?

Reply:
Yeah, @ethteck is going to do it after this is merged